tutorial/part2/chapter6.md

```diff
@@ -90,101 +92,93 @@ For this reason, we recommend importing the data before initialising the app, ri
-
-![Excel data 01](./ch6_files/data01.JPG)
-
 #### CSV files
-We will now upload the same data from above, but from a .csv file named [data_02](https://github.com/open-resources/dash_curriculum/blob/main/tutorial/part2/ch6_files/data_02.csv). Please follow the link and click the "Copy raw contents" button (next to the pencil edit button), then paste the records into a new Notepad file (Windows) or TextEdit file (MacOS), and save it as "data_02.csv". Find the path of your file to update the `filename` in the code below.
+
+We will now read in the same data from above, but from a .csv file named [data_02](https://github.com/open-resources/dash_curriculum/blob/main/tutorial/part2/ch6_files/data_02.csv). Please follow the link and click the "Raw" button in the top right corner of the file, right click the new page that opened and select "Save page as"/"Save as" and save it as "data_02.csv". Find the path of your file to update the `filename` in the code below.
 
 ```
-import pandas as pd
 filepath = r'C:\Users\User1\Downloads\data_02.csv'
 col_names = ['country','continent','year','pop']
 df2 = pd.read_csv(filepath, sep='|', usecols=col_names)
-print(df2.head())
 ```
 
-- Similarly to the previous example, we have accessed the data located outside the app folder, and therefore specified a filepath as a raw string.
-- In most cases, you will see .csv files with comma column separators (hence the name, csv standing for "comma-separated values"). Here, we used a different separator: '|'. The `sep` argument allows to specify whatever characters should be considered field separators: pandas will separate data into different columns any time it encounters these characters.
+- Similarly to the previous example, we have accessed the data located outside the app folder, and therefore specified a full path to the file.
+- In most cases, you will see `.csv` files with comma column separators (hence the name, csv standing for "comma-separated values"). Here, we used a different separator: '|'. The `sep` argument allows to specify whatever characters should be considered field separators: pandas will separate data into different columns any time it encounters these characters.
 - We have also selected a subset of columns to be uploaded, listed in the `usecols` argument. The remaining columns that are present in the file will be ignored.
 
+The data looks the same whether it is read from a CSV or Excel file:
+
 ![csv data 02](./ch6_files/data02.JPG)
 
 ### 6.2.2 Reading data from a URL
-We will now upload the same data from above, but from [this ULR](https://raw.githubusercontent.com/open-resources/dash_curriculum/main/tutorial/part2/ch6_files/data_03.txt).
+
+Instead of manually downloading the file first, we could read it from the URL directly:
 
 ```
-import pandas as pd
 url = 'https://raw.githubusercontent.com/open-resources/dash_curriculum/main/tutorial/part2/ch6_files/data_03.txt'
-df3 = pd.read_table(url, sep=';')
```

Comment:
I don't think there is any advantage of using `read_table` over `read_csv` but correct me if I am wrong